### ISSUE TITLE
terminal_bar.py: from __future__ import print_function

### ISSUE THREAD

**Issue by jankatins** [yoavram/ipy-progressbar#1]
Without that line, I get a syntax error on import: 

```
  File "C:\portabel\miniconda\envs\sc\lib\site-packages\ipy_progressbar\terminal_bar.py", line 25
    print(s, end=' ')
                ^
SyntaxError: invalid syntax
```

**Comment by yoavram** [comment]
Please note that this is just a mirror repo of the original at  [Alexander Plavin's repo](https://hg.plav.in/my_projects/python/ipy-progressbar). I'm will leave this and other issues open here but will not fix them.

**Comment by yoavram** [comment]
Fixed in 4dc6e62.
